src/stores/connectors/keys/import.go

```diff
@@ -3,14 +3,16 @@ package keys
 import (
 	"context"
 
+	entities2 "github.com/consensys/quorum-key-manager/src/entities"
```

Comment:
idem

Reply:
entities are now split in the different domains but we have a tech debt to move them to a single folder: https://app.zenhub.com/workspaces/tanuki-5ea70772b186e10067f57842/issues/consensys/quorum-key-manager/431